server/shared/publishing/publishing.service.js

```diff
@@ -15,17 +15,18 @@ class PublishingService {
     this.queue = new PromiseQueue(1, Infinity);
   }
 
+  async publish(action, payload) {
```

Comment:
Do you think we could find a name that better describes this method since it doesn't really do publishing specifically (it just ensures that the webhook is called after action)?

Or it could return a function (i.e. `toPublishJob`) so we can get rid of the arrows in the `queue.add` call. :bowtie:

Reply:
Also, should we make it private?